### PR TITLE
python.pkgs wide: fix problematic urls

### PR DESCRIPTION
**Pull request by jonringer**
###### Motivation for this change
noticed a bunch that are problematic on [repology](https://repology.org/repository/nix_unstable/problems)

python packages get a triple hit since there's a unique package per interpreter (2.7, 3.7, 3.8)

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

```
Nothing changed
https://github.com/NixOS/nixpkgs/pull/77468
$ nix-shell /home/jon/.cache/nix-review/pr-77468/shell.nix

[nix-shell:~/.cache/nix-review/pr-77468]$
```

### CODE REVIEW

**Comment by jonringer** [comment]
going to merge once ofborg's checkMeta finishes

File: pkgs/development/python-modules/Pygments/default.nix [hunk 1]
```diff
@@ -19,7 +19,7 @@ buildPythonPackage rec {
   doCheck = false;
 
   meta = {
-    homepage = http://pygments.org/;
+    homepage = https://pygments.org/;
```
Comment (alyssais) [inline pkgs/development/python-modules/Pygments/default.nix]: Might as well quote these while we’re here, per NixOS/rfcs#45.

Comment (jonringer) [reply pkgs/development/python-modules/Pygments/default.nix]: If i knew about this beforehand, i would have. Didn't realize it was accepted yet

Comment (jonringer) [reply pkgs/development/python-modules/Pygments/default.nix]: the quoting can also happen automatically by script, I'm not too concerned about it.

**Pull request merged**